internal/cli/atlas/clusters/onlinearchive/create.go

```diff
@@ -114,6 +131,14 @@ To learn more about online archives, see https://www.mongodb.com/docs/atlas/onli
   # Create an online archive for the sample_mflix.movies collection in a cluster named myTestCluster using a profile named egAtlasProfile when the current date is greater than the value of the released date plus 2 days. Data is partitioned based on the title field, year field, and released field from the documents in the collection:
   %[1]s clusters onlineArchive create --clusterName myTestCluster --db sample_mflix --collection movies --dateField released --archiveAfter 2 --partition title,year --output json -P egAtlasProfile `, cli.ExampleAtlasEntryPoint()),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if opts.filename == "" {
+				_ = cmd.MarkFlagRequired(flag.ClusterName)
+				_ = cmd.MarkFlagRequired(flag.Database)
+				_ = cmd.MarkFlagRequired(flag.Collection)
+				_ = cmd.MarkFlagRequired(flag.DateField)
+				_ = cmd.MarkFlagRequired(flag.ArchiveAfter)
+			}
```

Comment:
this just got released, [MarkFlagsOneRequired](https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#flag-groups) just got released and seems like this could be a usage for it, wanna give it a try?

Reply:
I've tested it and it works well if combined with `MarkFlagsRequiredTogether`